x/payments/internal/types/params.go

```diff
@@ -133,18 +97,218 @@ func (p Params) String() string {
 	)
 }
 
+func validateFactor(i interface{}) error {
+	v, ok := i.(sdk.Dec)
+
+	if !ok {
+		return fmt.Errorf("invalid parameter type: %T", i)
+	}
+
+	if v.IsNil() {
+		return fmt.Errorf("ixo factor must be not nil")
+	}
+	if v.IsNegative() {
+		return fmt.Errorf("ixo factor must be positive: %s", v)
+	}
+	if v.GT(sdk.OneDec()) {
+		return fmt.Errorf("ixo factor too large: %s", v)
+	}
+
+	return nil
+}
+
+func validateFeeAmount(i interface{}) error {
+	v, ok := i.(sdk.Dec)
+
+	if !ok {
+		return fmt.Errorf("invalid fee amount type: %T", i)
+	}
+
+	if v.IsNil() {
+		return fmt.Errorf("fee amount must be not nil")
+	}
+	if v.IsNegative() {
+		return fmt.Errorf("fee amount must be positive: %s", v)
+	}
+	if v.GT(sdk.OneDec()) {
```

Comment:
No need to add new checks. The checks that there were before for FeeAmount (just check that it is positive) are enough. Blockchain not starting because of this.

Reply:
removed